File: problemas/1000/1079.dart
import 'dart:io';

void main() {
  final n = int.parse(stdin.readLineSync());

  for (int i = 0; i < n; i++)
  {
    final linha = stdin.readLineSync().split(' ');
    final a = double.parse(linha[0]);
    final b = double.parse(linha[1]);
    final c = double.parse(linha[2]);

    stdout.writeln(((a * 2 + b * 3 + c * 5) / 10.0).toStringAsFixed(1));
  }
}

File: problemas/1000/1079.cs
using System;

namespace uri1079
{
    internal static class Program
    {
        private static void Main()
        {
            int n = int.Parse(Console.ReadLine());

            for (int i = 0; i < n; i++)
            {
                string[] linha = Console.ReadLine().Split(' ');
                double a = double.Parse(linha[0]);
                double b = double.Parse(linha[1]);
                double c = double.Parse(linha[2]);

                Console.WriteLine($"{(a * 2 + b * 3 + c * 5) / 10.0:F1}");
            }
        }
    }
}